presentation/src/main/java/com/yapp/presentation/model/TeamModel.kt

```diff
@@ -0,0 +1,9 @@
+package com.yapp.presentation.model
+
+import com.google.gson.annotations.SerializedName
+
+data class TeamModel(
+    @SerializedName("team")
+    val team: String = "",
```

Comment:
```suggestion
    val teamType: TeamType = "",
```
enum type 으로 ios, and, web team 을 정의하는게 좋지 않을까 싶어요!
처음에 team 이 string type 이었다 보니 코드 읽었을을 때 어떤걸 담아내는 아이인지 헷갈렸던 것 같아요.

Reply:
다른 기수 때 팀 이름이 변할 수 있지 않을까요? 이름만 `teamType`으로 바꾸는건 어떨까요!

Reply:
이 부분 합의 되면 저도 코드에 반영하겠습니다!

Reply:
일단 `teamType`과 `teamNum`으로 수정해뒀습니당